File: services/delete_user.py
from models.user import User
from database import Base, engine, SessionLocal

def delete_user(email: str):
    Base.metadata.create_all(bind=engine)
    db = SessionLocal()
    user = db.query(User).filter_by(email=email).first()

    if user:
        db.delete(user)
        db.commit()
        print("🗑️ Utilisateur supprimé :", email)
        return True
    else:
        print("❌ Utilisateur non trouvé :", email)
        return False



if __name__ == "__main__":
    user = delete_user("test@hello.com")

File: services/delete_user.js
// services/delete_user.js
const User = require('../models/User');

function deleteUser(email) {
  const user = User.findByEmail(email);
  
  if (user) {
    User.deleteByEmail(email);
    console.log("🗑️ Utilisateur supprimé :", email);
    return true;
  } else {
    console.log("Utilisateur non trouvé :", email);
    return false;
  }
}

module.exports = deleteUser;

